### ISSUE TITLE
react-native-settings under Expo  - keyboard gets in the way of the text input trying to edit

### ISSUE THREAD

**Issue by geetee24** [mmomtchev/react-native-settings-screen#83]
Sample app I cant edit the field for it pops up the keyboard which covers the field I am trying to edit and hence cant see the data I am entering.

**Comment by mmomtchev** [comment]
can I see the mobile application your using it with?

**Comment by geetee24** [comment]
trying to edit a field hidden under the keyboard that appeared.
![Image](https://github.com/mmomtchev/react-native-settings/assets/6679119/0c7c8c21-b7dc-4322-beff-80e52dad0143)

**Comment by geetee24** [comment]
perhaps you can wrap your <TextInput> with this lib.
https://github.com/APSL/react-native-keyboard-aware-scroll-view

also - if the settings page has too many elements in it, it wont scroll up to allow all the fields to be edited.

**Comment by mmomtchev** [comment]
This issue is used for simultaneous posts related to my extortion involving the French police. Is there a criminal message that you wish to convey?

**Comment by mmomtchev** [comment]
@oniatsu  But if this plan is adopted, once the application is redeployed, will the cache be reset or cleared?

_Originally posted by @EryouHao in https://github.com/vercel/next.js/discussions/48324#discussioncomment-9260842_

**Comment by mmomtchev** [comment]
_Originally posted by @mmomtchev in https://github.com/conan-io/conan-center-index/issues/23789#issuecomment-2082476231_

**Comment by mmomtchev** [comment]
Likely a duplicate of #23698

_Originally posted by @Ahajha in https://github.com/conan-io/conan-center-index/issues/23789#issuecomment-2081677791_

**Comment by geetee24** [comment]
Not a duplicate

**Comment by mmomtchev** [comment]
No, this was one of the messages that were posted simultaneously with yours for the extortion that you are trying to make it look as if you didn't notice

**Comment by geetee24** [comment]
huh? I dont understand what you are saying.  Do you think you can fix the issues I raised?  Thanks much

**Comment by Ahajha** [comment]
I've also been dragged into this and have no idea what I did. I have no clue what extortion he's talking about. I also have nothing to do with the French police (I don't even live in Europe, never have even been there), so I think he's just reading _way_ too far into some completely innocent comments.

**Comment by geetee24** [comment]
I too have no clue.  I just hoping the issue I raised can get looked at.   - best to you.  Thanks

**Comment by mmomtchev** [comment]
Both of you and a third person made simultaneous comments linked to the extortion that I have been living with.

**Comment by Ahajha** [comment]
Okay, maybe we made "simultaneous comments", fine, coincidence. How in the hell do any of those comments have _anything_ to do with your problem? Literally all I did was link a duplicate issue about a build issue on Windows in debug mode regarding a text library. I'm almost curious how you're pinning this coincidence to the French Police or your extortion problem?

**Comment by mmomtchev** [comment]
I have been living in total isolation for the last three years, since my ex-employers reached a deal with lawyers and judges to cover up a series of serious crimes. They were hoping that they were going to reach the same deal with any future employer I would have to keep this under wraps. By now, the affair is absolutely everywhere, and for the last 6 months I never had a single comment or issue not related to the extortion.

**Comment by mmomtchev** [comment]
6 months ago the police tried to intimidate me by coming to search my home illegally. They held me at gunpoint, then when they realized that this was not going to work, they simply left. From this day on, I have been getting only simultaneous comments. Yours was one of three that came at the same time.

**Comment by mmomtchev** [comment]
In the legal world there is a notion of a "guilt proven beyond reasonable doubt". I think that your case fits this description perfectly.

**Comment by Ahajha** [comment]
Think what you want buddy, if you think that literally everything revolves around this case that I've never heard of then you're delusional, flat out. You need some help. I don't know how to make it clear to you that not only have I never heard of this, but I couldn't even possibly be in connection with the people that wronged you. I live and have always lived on a different continent. Hell, 3 years ago I was in college.

The world doesn't revolve around you.

**Comment by mmomtchev** [comment]
For me you are a cheeky criminal.

**Comment by mmomtchev** [comment]
And by the way, I would just like to point out that the nature of this criminal affair is such, as to warrant a psychiatric evaluation for everyone taking part in it. A real one.
